hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystem.java

```diff
@@ -1750,6 +1753,67 @@ public void testUnbuffer() throws IOException {
 
   }
 
+  /**
+   *  Test EC checksum with Replicated checksum.
+   */
+  @Test
+  public void testFileChecksum() throws IOException {
+    int dataLen = 1024 * 1024 * 5;
+    byte[] data = RandomStringUtils.randomAlphabetic(dataLen)
+        .getBytes(UTF_8);
+
+    BucketArgs.Builder builder = BucketArgs.newBuilder();
+    builder.setStorageType(StorageType.DISK);
+    builder.setBucketLayout(BucketLayout.LEGACY);
+    builder.setDefaultReplicationConfig(
+        new DefaultReplicationConfig(ReplicationType.EC,
+            new ECReplicationConfig("RS-3-2-1024k")));
+    BucketArgs omBucketArgs = builder.build();
+
+    String vol = UUID.randomUUID().toString();
+    String ecBucket = UUID.randomUUID().toString();
+    final OzoneBucket bucket101 = TestDataUtil
+        .createVolumeAndBucket(cluster, vol, ecBucket, BucketLayout.LEGACY,
+            omBucketArgs);
+
+    Assert.assertEquals(ReplicationType.EC.name(),
+        bucket101.getReplicationConfig().getReplicationType().name());
+
+    try (OzoneFSOutputStream file = adapter
+        .createFile(vol + "/" + ecBucket + "/test", (short) 3, true, false)) {
+      file.write(data);
+    }
+
+    Path parent = new Path("/" + vol + "/" + ecBucket + "/");
+    Path ecKey = new Path(parent, "test");
+    FileChecksum ecChecksum = fs.getFileChecksum(ecKey);
+    String ecChecksumString = StringUtils.byteToHexString(
+        ecChecksum.getBytes(), 0, ecChecksum.getLength());
+
```

Comment:
Nit: Below: you can create in single statement.
BucketArgs omBucketArgs = BucketArgs.newBuilder().setStorageType(StorageType.DISK).setBucketLayout(BucketLayout.LEGACY).build();
?